recipe/meta.yaml

```
@@ -61,14 +61,8 @@ requirements:
     - {{ pin_compatible('numpy') }}
 
 {% set tests_to_skip = "_not_a_real_test" %}
-# skip some tests that hang/fail on pypy
-{% if python_impl == "pypy" %}
-  # hangs
-  {% set tests_to_skip = tests_to_skip + " or test_mixed_threads_processes" %}  # [aarch64 or ppc64le]
-  # failures
-  {% set tests_to_skip = tests_to_skip + " or test_cdist_refcount" %}           # [unix]
-  {% set tests_to_skip = tests_to_skip + " or test_boost_eval_issue_14606" %}   # [unix]
-{% endif %}
+{% set tests_to_skip = " or test_cdist_refcount" %}          # [python_impl == "pypy"]
+{% set tests_to_skip = " or test_boost_eval_issue_14606" %}  # [python_impl == "pypy"]
```

I think there is a typo here
```suggestion
{% set tests_to_skip = tests_to_skip + " or test_cdist_refcount" %}          # [python_impl == "pypy"]
{% set tests_to_skip = tests_to_skip + " or test_boost_eval_issue_14606" %}  # [python_impl == "pypy"]
```

---

Woops, sorry about this 😅